lib/lev/routine.rb

```diff
@@ -397,7 +402,11 @@ def errors?
     end
 
     def fatal_error(args={})
-      errors.add(true, args)
+      if self.class.raise_fatal_errors?
```

Comment:
In the case of Routine A calling Routine B and a fatal_error occurring in Routine B, I wonder if it should be up to Routine A to say whether or not fatal errors should be exceptions...  thinking of the case of a handler calling a routine and wanting to not raise exceptions. 

In which case this line would be

```ruby
if topmost_runner.class.raise_fatal_errors?
```

`topmost_runner` already exists to climb the containment tree and get the routine at the top.  Thoughts @joemsak?